domains/POAS/events/lpa-updated/schema.json

```diff
@@ -0,0 +1,18 @@
+{
+  "$id": "https://opg.service.justice.gov.uk/opg.poas.lpastore/lpa-updated.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "opg.poas.lpastore/lpa-updated",
+  "type": "object",
+  "properties": {
+    "uid": {
+      "type": "string",
+      "description": "The UID of the LPA",
+      "pattern": "M(-[A-Z0-9]{4}){3}"
+    },
+    "createdAt": {
```

Comment:
I think changeType is missing from the properties. I was going to suggest changeType should be an enumeration, but at the moment we don't know exactly which types of change we will need, so it's OK to leave it for now.

Reply:
Thanks for catching this